lib/ruby_lsp/requests/code_lens.rb

```diff
@@ -263,6 +282,28 @@ def add_open_gem_remote_code_lens(node, remote)
           data: { type: "link" },
         )
       end
+
+      sig { params(node: Prism::CallNode, kind: Symbol).void }
+      def add_spec_code_lens(node, kind:)
+        first_argument = T.must(node.arguments).arguments.first
+        return unless first_argument
+
+        method_name = case first_argument
+        when Prism::StringNode
+          first_argument.content
+        when Prism::ConstantReadNode
+          first_argument.full_name
+        end
+
+        return unless method_name
+
+        add_test_code_lens(
+          node,
+          name: method_name,
+          command: generate_test_command(class_name: method_name),
```

Comment:
To be honest, I'm not fully satisfied with this `class_name: method_name` discrepancy. This is working, but this is clearly not a `class_name`. The fact is, for spec-style, there is no class. Any suggestion / challenge are welcome.